<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lib</class>
 <widget class="QDialog" name="lib">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>547</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>793</width>
    <height>547</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>793</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="search_edit">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>10</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="search_button">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Go</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/48/search.png</normaloff>image/48/search.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QTreeWidget" name="tree">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>38</y>
     <width>471</width>
     <height>501</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Book</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Author</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="borrow_button">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>490</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Borrow books</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/48/shield-ok.png</normaloff>image/48/shield-ok.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>40</y>
     <width>291</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Details</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QLineEdit" name="book_name">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="book_author">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="book_prize">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="book_remain">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Author</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Prize</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Remain</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="book_des">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>330</y>
     <width>291</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Book Cart</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QTreeWidget" name="cart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>2</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="add_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to Cart</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>image/48/box-in.png</normaloff>image/48/box-in.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove 
from Cart</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>image/48/box-out.png</normaloff>image/48/box-out.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="info_button">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/48/sign-info.png</normaloff>image/48/sign-info.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="sign_out">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign out</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/numix_circle_for_windows_by_neiio-dag5dlc/Numix Circle For Windows/PNG/kshutdown.png</normaloff>image/numix_circle_for_windows_by_neiio-dag5dlc/Numix Circle For Windows/PNG/kshutdown.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="admin">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Admin</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/48/user-male.png</normaloff>image/48/user-male.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="library">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Library</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/numix_circle_for_windows_by_neiio-dag5dlc/Numix Circle For Windows/PNG/calibre-ebook-edit.png</normaloff>image/numix_circle_for_windows_by_neiio-dag5dlc/Numix Circle For Windows/PNG/calibre-ebook-edit.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="history">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>History</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/numix_circle_for_windows_by_neiio-dag5dlc/Numix Circle For Windows/PNG/calibre.png</normaloff>image/numix_circle_for_windows_by_neiio-dag5dlc/Numix Circle For Windows/PNG/calibre.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="contact">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>490</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Contact</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/numix_circle_for_windows_by_neiio-dag5dlc/Numix Circle For Windows/PNG/gmail-offline.png</normaloff>image/numix_circle_for_windows_by_neiio-dag5dlc/Numix Circle For Windows/PNG/gmail-offline.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="message">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>490</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Message</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/48/bubble.png</normaloff>image/48/bubble.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="lib_message">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>490</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Message</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/48/bubble.png</normaloff>image/48/bubble.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="request">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>490</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Request</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/48/bullhorn.png</normaloff>image/48/bullhorn.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>551</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="design.qrc">:/new/image/pink.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Search for:</string>
   </property>
  </widget>
  <zorder>label_5</zorder>
  <zorder>search_edit</zorder>
  <zorder>search_button</zorder>
  <zorder>tree</zorder>
  <zorder>borrow_button</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>info_button</zorder>
  <zorder>sign_out</zorder>
  <zorder>admin</zorder>
  <zorder>library</zorder>
  <zorder>history</zorder>
  <zorder>contact</zorder>
  <zorder>message</zorder>
  <zorder>lib_message</zorder>
  <zorder>request</zorder>
  <zorder>label_6</zorder>
 </widget>
 <resources>
  <include location="design.qrc"/>
 </resources>
 <connections/>
</ui>
